File: response.py
import os

query = os.environ.get('QUERY_STRING')

content = "<!DOCTYPE html> \
<html lang=\"en\"> \
    <head> \
        <link href=\"./style/style.css\" rel=\"stylesheet\"> \
        <link href=\"./style/form.css\" rel=\"stylesheet\"> \
        <title>Response</title> \
    </head> \
    <body> \
        <h1 class=\"title\">Response</h1> \
        <h2>Hello, World!</h2> \
        <p>" + query + "</p> \
        <a href=\"site_index.html\"><button>INDEX</button></a> \
    </body> \
</html>"

print(content)

File: cgi-bin/response.php
<?php
    $query = $_SERVER["QUERY_STRING"];
    $params = array();
    parse_str($query, $params);
    echo '<!DOCTYPE html>
    <html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link href="/style/style.css" rel="stylesheet">
        <link href="/style/response.css" rel="stylesheet">
        <title>Response</title>
    </head>
    <body>
        <h1 class="pagesTitle">Response</h1>
        <div class="response">
            <div class="tag">Name : <span class="value">' .$params['name'] . '</span></div>
            <div class="tag">Firstname : <span class="value">' .$params['firstname'] . '</span></div>
            <div class="tag">Login : <span class="value">' .$params['login'] . '</span></div>
        </div>
        <div class="index">
            <a class="indexButton" href="/site_index.html">Index</a>
        </div>
    </body>
    </html>';
?>